docs/checkers/index.md

```diff
@@ -2699,6 +2699,48 @@ python_ta.check_all(config={
 })
 ```
 
+(R9900)=
+
+### Redundant Condition (R9900)
+
+This error occurs when an `if` or `while` condition is guaranteed to be true.
+
+Example:
+
+```{literalinclude} /../examples/custom_checkers/r9900_redundant_condition.py
+
+```
+
+This error will only be checked if `z3` dependency is install and `z3` option of PythonTA
```

Comment:
`z3` dependency -> the `z3-solver` library

install -> installed

is turned on -> is enabled